lang/main.json

```diff
@@ -470,16 +466,8 @@
     },
     "invite": {
         "addPassword": "Add password",
-        "callNumber": "Call __number__",
-        "enterID": "Enter Meeting ID: __conferenceID__ following by # to dial in from a phone",
-        "howToDialIn": "To dial in, use one of the following numbers and meeting ID",
-        "hidePassword": "Hide password",
         "inviteTo": "Invite people to __conferenceName__",
-        "invitedYouTo": "__userName__ has invited you to the __inviteURL__ conference",
-        "invitePeople": "Invite",
-        "locked": "This call is locked. New callers must have the link and enter the password to join.",
-        "showPassword": "Show password",
-        "unlocked": "This call is unlocked. Any new caller with the link may join the call."
+        "invitePeople": "Invite"
```

Comment:
Hm where do we use this one? I don't see it in the code, but maybe I'm missing something?